docs/developers.rst

```diff
@@ -0,0 +1,43 @@
+Developers
+==========
+
+The installation and initialiseation steps are slightly different if you want to work on pytac.
+
+
+Installation
+~~~~~~~~~~~~
+
+This is only required on your first use.
+
+- Ensure you have the following requirements: Pip, Pipenv, and a local copy of pytac.
+
+- Install dev-packages, pytac, and cothread for EPICS support::
+
+    $ pipenv install --dev
+    $ pip install pytac
```

Comment:
You don't need to install pytac here because you will be using the version in the repo. This is why you got the strange behaviour when you were trying to build the documentation.